cscs-checks/apps/namd/namd_check.py

```diff
@@ -42,6 +48,21 @@ def __init__(self, arch, scale, variant):
                 self.num_tasks = 16
                 self.num_tasks_per_node = 1
 
+    @rfm.run_after('init')
+    def set_valid_prgenv(self):
+        if self.current_system.name == 'pilatus':
+            self.valid_prog_environs = ['cpeIntel']
+        else:
+            self.valid_prog_environs = ['builtin']
```

Comment:
I would put all the valid prog. environments at the beginning and, here, I would remove those that are not valid. This makes the test, I think, more readable.

Reply:
Done